src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ValidationService.cs

```diff
@@ -59,17 +59,21 @@ public async Task<bool> ValidatePublishingState(Guid releaseId)
             if (scheduled != null)
             {
                 _logger.LogError(
-                    $"Validating {ValidationStage.ReleasePublishingStateNotScheduledOrStarted.ToString()} failed: " +
-                    $"Publishing is already scheduled. ReleaseStatus: {scheduled.Id}");
+                    "Validating {ReleasePublishingStateNotScheduledOrStartedString} failed: " +
```

Comment:
It might have been cleaner and clearer if you'd left the arguments in the template as `{0}`, `{1}`, `{2}`, etc throughout all these changes rather than things like `{MessageReleaseId}` and `{ReleasePublishingStateNotScheduledOrStartedString}` since the names don't matter at but this gives a hint to the parameter order.